pkg/antctl/raw/check/cluster/command.go

```diff
@@ -101,6 +101,9 @@ func Run(o *options) error {
 	}
 	ctx := context.Background()
 	testContext := NewTestContext(client, config, clusterName, o.testImage)
+	defer func() {
+		check.Teardown(ctx, testContext.client, testContext.clusterName, testContext.namespace)
+	}()
```

Comment:
no need to wrap the call to `check.Teardown` with a function

Reply:
Updated.